docs/modules/ROOT/pages/backup-restore.adoc

```diff
@@ -112,7 +112,7 @@ For the full list of the supported expression, please check the link:https://pkg
 +
 [source,bash]
 ----
-kubectl apply -f ./hot-backup-scheduled.yaml
+kubectl apply -f ./cron-hot-backup.yaml
 ----
 
 === Checking the Status of a Backup
```

Comment:
```suggestion
=== Checking the Status of a Local Backup
```
Also, should we say.... "To check the status of a local backup, run the following command:". Also do we need to provide an example to show how to check the status of a local scheduled backup/is it the same? Can you check the status of a specific scheduled backup, i.e. by filename?